### PR TITLE
Convert global 'wpt' namespacing to es6 imports

### PR DESCRIPTION
**Pull request by lukebjerring**
## Description
Cleans up the code a lot :)

### CODE REVIEW

File: webapp/components/display-logo.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ class DisplayLogo extends ProductInfo(PolymerElement) {
     if (!showSource || !product.labels) {
       return '';
     }
-    const sources = window.wpt.Sources;
+    const sources = Sources;
     return product.labels.find(s => sources.has(s));
```
Comment (LukeZielinski) [inline webapp/components/display-logo.js]: do you need the sources var (ie: can you just do Sources.has(s) below)?

File: webapp/components/test-runs-query-builder.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ import '../node_modules/@vaadin/vaadin-date-picker/vaadin-date-picker-light.js';
 import '../node_modules/@vaadin/vaadin-date-picker/vaadin-date-picker.js';
 import './display-logo.js';
 import './info-banner.js';
-import { ProductInfo } from './product-info.js';
+import { ProductInfo, DefaultBrowserNames, DefaultProducts, Channels, SemanticLabels } from './product-info.js';
```
Comment (LukeZielinski) [inline webapp/components/test-runs-query-builder.js]: nit: alphabetical order

File: webapp/components/display-logo.js (hunk 1 above)
Comment (lukebjerring) [reply webapp/components/display-logo.js]: Nope; removed.

File: webapp/components/display-logo.js [hunk 3]
```diff
@@ -116,7 +116,7 @@ class DisplayLogo extends ProductInfo(PolymerElement) {
     if (!showSource || !product.labels) {
       return '';
     }
-    const sources = window.wpt.Sources;
+    const sources = Sources;
```
Comment (LukeZielinski) [inline webapp/components/display-logo.js]: Do you need the sources var (ie: can you just do Sources.has(s) below)?

File: webapp/components/test-runs-query-builder.js (hunk 2 above)
Comment (lukebjerring) [reply webapp/components/test-runs-query-builder.js]: Done.

File: webapp/components/test-runs-query.js [hunk 4]
```diff
@@ -3,17 +3,10 @@
  * Use of this source code is governed by a BSD-style license that can be
  * found in the LICENSE file.
  */
-
-import { ProductInfo } from './product-info.js';
+import { ProductInfo, DefaultProducts, DefaultProductSpecs, Channels } from './product-info.js';
```
Comment (LukeZielinski) [inline webapp/components/test-runs-query.js]: nit: alphabetical

File: webapp/components/display-logo.js (hunk 3 above)
Comment (lukebjerring) [reply webapp/components/display-logo.js]: Nope; fixed.

**Pull request merged**